spec/integrations/users_show_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe 'users show', type: :system do
+  let(:user) { User.new(name: 'Diego', bio: 'Hi I\'m a student from Mexico', photo: 'https://as1.ftcdn.net/v2/jpg/00/64/30/06/1000_F_64300613_1TYCvyzq09Go7oEVXkDQypofuaJEwKTo.jpg') }
+  let(:post1) { Post.new(user:, title: 'Post 1', text: 'Post content') }
+  let(:post2) { Post.new(user:, title: 'Post 2', text: 'Post content') }
+  let(:post3) { Post.new(user:, title: 'Post 3', text: 'Post content') }
+  let(:post4) { Post.new(user:, title: 'Post 4', text: 'Post content') }
+  let(:comment1) { Comment.new(user:, post: post1, text: 'Comment 1') }
+  let(:comment2) { Comment.new(user:, post: post2, text: 'Comment 2') }
+  let(:comment3) { Comment.new(user:, post: post3, text: 'Comment 3') }
+  let!(:like1) { Like.create(user:, post: post1) }
+
+  before :each do
+    user.save
+    post1.save
+    post2.save
+    post3.save
+    post4.save
+    comment1.save
+    comment2.save
+    comment3.save
+  end
+
+  it 'displays the profile picture for each user' do
+    visit users_path(user)
+    expect(page).to have_css('img.user-photo')
+  end
```

Comment:
- [ ] Dear team good job updating your test as suggested by the previous reviewer, but still your test does not assert the appropriate profile picture is visible. Please make sure the `user.photo` or the link for the image `https://as1.ftcdn.net/v2/jpg/00/64/30/06/1000_F_64300613_1TYCvyzq09Go7oEVXkDQypofuaJEwKTo.jpg` is present in the page. 
- [Optional] Also consider updating the description for this test. we are only testing a single user profile picture, hence kindle remove/update the `for each user` part od the `it` blcok.